src/components/Note/NoteAuthorList/NoteAuthorList.scss

```diff
@@ -0,0 +1,105 @@
+@import "src/constants/style.scss";
+
+.note-author-list {
+  display: flex;
+  flex-direction: row-reverse;
+  align-items: center;
+}
+
+.note__author {
+  display: inline-flex;
+  align-items: center;
+  // reset css from figure-element:
+  margin: 0 0 0 #{-$margin--medium}; // overlap
+  border: 0;
+  font: inherit;
+  vertical-align: baseline;
+  flex-grow: 0;
+  height: 28px;
+  padding: 0 $padding--default 0 2px;
+  background: $color-white;
+  border-radius: $rounded--default;
+}
+
+.note-author-rest__wrapper {
+  padding: 2px;
+  border-radius: $rounded--default;
+}
+
+.note-author-rest__wrapper--self {
+  background: var(--accent-color--desaturated-light);
+}
+
+.note-author-rest {
+  display: inline-flex;
+  width: $icon--medium;
+  height: $icon--medium;
+
+  margin: 0 $margin--default 0 2px;
+
+  font-size: $text-size--small;
+
+  color: $color-white;
+  background-color: $menu-icon-background-color--dark;
+
+  justify-content: center;
+  align-items: center;
+
+  border: 1px solid $color-white-one;
+  border-radius: $rounded--full;
+}
+
+// actually affects the first one, since the flex direction is reversed
+// also we only want to change the margin if no rest users exist, since they are displayed on the left
+.note__author:not(.note__author--with-rest):last-of-type {
+  margin-left: 0;
+}
+
+.note__author-name {
+  margin-left: $margin--small;
+  color: $color-dark-gray;
+  font-size: $text-size--small;
+  letter-spacing: $letter-spacing--small;
+  line-height: $line-height--small;
+}
+
+.note__author--self {
+  background: var(--accent-color--desaturated-light);
+}
+
+.note__author--self > .note__user-avatar #Circle-Background {
+  stroke: none;
```

Comment:
imo it'd be nice to see if my own note is somewhere inside of a stack

Reply:
Yeah, I was considering to always display yourself as of of the avatars instead of the rest author list, independently of the position in the stack. Would that work for you?

Reply:
sounds good, lets try that

Reply:
yourself should always be visible now: ecaa612